src/elements/hx-toast/_shadow.scss

```diff
@@ -31,8 +35,11 @@
 }
 
 #hxCta {
-  &:extend(.hxBtn);
-  &:extend(.hxBtn.hxTertiary);
+  @include button-base;
+  @include button-medium;
+  // NOTE: button-tertiary is bringing in more than necessary, but it won't be
+  // an issue once we refactor Toast to define CTA elements in Light DOM.
+  @include button-tertiary;
```

Comment:
It's worth noting that with the updates to using mixins, tertiary mixin styles had additional CSS that wasn't originally provided. These styles are beneficial, though, because they add styles for different states of the CTA button.  However, when we get around to refactoring the Toast component to defined the CTA in the Light DOM, this extra CSS will be unnecessary and can be removed from the Shadow DOM styles.

Reply:
@araiford: Here's the update that added additional styles to the Toast CTA.